falcon/testing/resource.py

```diff
@@ -108,11 +109,20 @@ def __init__(self, status=None, body=None, json=None, headers=None):
         else:
             self._default_body = body
 
+        self.captured_req = None
+        self.captured_resp = None
+        self.captured_kwargs = None
+
     @falcon.before(capture_responder_args)
     @falcon.before(set_resp_defaults)
     def on_get(self, req, resp, **kwargs):
         pass
 
+    @falcon.before(capture_responder_args)
+    @falcon.before(set_resp_defaults)
+    def on_post(self, req, resp, **kwargs):
```

Comment:
Needed this since the PostQueryParams tests are now required to use POST instead of GET.